waterloo-rocketry/website-react
# Shift backgroundPosition of cover photos

## Description

As a result of #53, the cropping of the cover pages on a lot of the pages became cut off due to their heights being reduced. This PR aims to fix this by moving around the `backgroundPosition` for them. I also increased the heights of the cover photos slightly to allow them to show off more stuff.




## Developer Testing

Testing done:
- Manual visual inspection

## Reviewer Testing


Steps to view/test:
1. `git clone` this repo
1. `npm install`
1. `npm start`
1. Take a look at the cover pages on each page and ensure that they looks as intended


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/waterloo-rocketry/website-react/57)
